### ISSUE TITLE
Error on new protocol (Trac #99)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#103]
The following error occurs on startup or when a new protocol is created from the menu "File/Protocol/New".

ERROR: microdrop.gui.protocol_grid_controller plugin crashed processing on_protocol_created signal.
Reason: type object argument after ** must be a mapping, not NoneType

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/99
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "The following error occurs on startup or when a new protocol is created from the menu \"File/Protocol/New\".\n\nERROR: microdrop.gui.protocol_grid_controller plugin crashed processing on_protocol_created signal.\nReason: type object argument after ** must be a mapping, not NoneType", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Error on new protocol", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-04-17T19:02:06", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120417T19:40:09: ryan commented:

> This error only shows up if the dmf_control_board plugin is enabled.

**Comment by cfobel** [comment]
Trac update at 20120417T23:20:20: 
* christian@fobel.net commented:

> commit 0d0c1074306e6033934f4632bf5801d8cdcadf55
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Apr 17 19:11:27 2012 -0400
> 
>     Move insert/delete from buttons to popup menu
>     
>     Fixes #105.  Note that this also fixes #103 and fixes #104 after updating the
>     dmf_control_board plugin to revision v0.1-216-g03278e8.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
